Earthfile

```diff
@@ -465,10 +465,10 @@ arm-image:
   ARG MODEL=rpi64
   ARG IMAGE_NAME=${FLAVOR}.img
   WORKDIR /build
-  ENV STATE_SIZE="6200"
-  ENV RECOVERY_SIZE="4200"
+  ENV STATE_SIZE="6900"
+  ENV RECOVERY_SIZE="4600"
   ENV SIZE="15200"
-  ENV DEFAULT_ACTIVE_SIZE="2000"
+  ENV DEFAULT_ACTIVE_SIZE="2300"
```

Comment:
I wonder if we can calculate this based on something during the build script to not have to manually set it up...